### PR TITLE
use adminemail from config.php file properly

### PR DESCRIPTION
**Pull request by dmorley**
My first try at a PR on own repo, I think this works or might be better to fork my own copy of the repo.

### CODE REVIEW

**Comment by noplanman** [comment]
Oh, you haven't rebased your develop branch, which is why there are conflicts.

**Comment by dmorley** [comment]
Ill try IMing you, I think I have a mess now :-1:

**Comment by noplanman** [comment]
Ok, have managed to squash everything and clean it up.
Not sure why this was such an operation 😕 

Merge at your leisure 👍

File: db/gettoken.php [hunk 1]
```diff
@@ -45,7 +45,7 @@
     $to      = 'support@diasp.org';
```
Comment (noplanman) [inline db/gettoken.php]: Here, change this guy too!

Comment (dmorley) [inline db/gettoken.php]: you found 1 or 2 more?

Comment (noplanman) [inline db/gettoken.php]: Just the 1

Comment (dmorley) [inline db/gettoken.php]: Hrm guessing I will make a mess if I try and push my branch that is a mess. I almost want to kill this PR and start over with a new branch and see if I can learneded this thing

Comment (noplanman) [inline db/gettoken.php]: Just pull the latest version of your branch and rebase, that should be fine. It should fetch the cleaned up version. Then you can commit your new change.

**Comment by dmorley** [comment]
Ok changes made

**Comment by noplanman** [comment]
Great! You can merge now :+1:

**Pull request merged**